content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -98,231 +103,246 @@ The latest tag of the OpenTelemetry Collector Contrib distro <a href="https://gi
 Pin the Collector to the latest version to pick up the latest changes.
 </div>
 
-#### Receive traces from host
+### Receive traces from host
 
-1. Create a `otel_collector_config.yaml` file. [Here is an example template](#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's OTLP receiver and the Datadog exporter.
+Follow these steps if you wish to run the OpenTelemetry Collector as a Docker image and receive traces from the same host:
 
-2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:latest`][10].
+1. Create a `collector.yaml` file. You may use the [example template above](#configuring-the-datadog-exporter) to get started.
 
-3.  Determine which ports to open on your container. OpenTelemetry traces are sent to the OpenTelemetry Collector over TCP or UDP on several ports, which must be exposed on the container. By default, traces are sent over OTLP/gRPC on port `4317`, but common protocols and their ports include:
+2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:latest`][10].
```

Comment:
As staged in the div above this section, we must not tell people to use `latest` because of open-telemetry/opentelemetry-collector-releases/issues/73. We should tell them to use the latest tagged version.

```suggestion
2. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib:<VERSION>`][10].
```